clrtrust

```diff
@@ -96,102 +99,88 @@ EOF
 }
 
 cmd_check() {
-    local opt_verbose
-    local ret # return value
-
-    ret=0
 
     while [ $# -gt 0 ]; do
         case $1 in
-            -h|--help)
+            ("-h"|"--help")
                 print_check_help
                 return 0
                 ;;
-            -v|--verbose)
-                opt_verbose=1
+            ("-v"|"--verbose")
+                VERBOSE=1
                 shift
                 ;;
-            *)
+            (*)
                 print_check_help
                 return $EINVAL
                 ;;
         esac
     done
 
-    if [ ! -z "${VERBOSE}" ]; then
-        opt_verbose=1
-    fi
-
     if [ -e "${CLR_LOCAL_TRUST_SRC}" ]; then
         if [ ! -d "${CLR_LOCAL_TRUST_SRC}" ]; then
             1>&2 echo "${CLR_LOCAL_TRUST_SRC} must be a directory."
-            test ! -z $opt_verbose && 1>&2 cat <<EOF
-    ${CLR_LOCAL_TRUST_SRC} is used by ${BASENAME} to store local (user-added) trust and
-    distrust data.
-EOF
-            ret=$EBADST
+            verbose_error       \
+"%4s${CLR_LOCAL_TRUST_SRC} is used by ${BASENAME} to store local (user-added)
+%4strust and distrust data."
+            return $EBADST
         fi
-        if [ -e "${CLR_LOCAL_TRUST_SRC}/distrusted" -a ! -d "${CLR_LOCAL_TRUST_SRC}/distrusted" ]; then
+        if [ -e "${CLR_LOCAL_TRUST_SRC}/distrusted" ] && [ ! -d "${CLR_LOCAL_TRUST_SRC}/distrusted" ]; then
             1>&2 echo "$CLR_LOCAL_TRUST_SRC/distrusted must be a directory."
-            test ! -z $opt_verbose && 1>&2 cat <<EOF
-    ${CLR_LOCAL_TRUST_SRC}/trusted is a directory used by ${BASENAME} to store
-    trusted root CA certificates.
-EOF
-            ret=$EBADST
+            verbose_error       \
+"%4s${CLR_LOCAL_TRUST_SRC}/trusted is a directory used by ${BASENAME} to store
```

Comment:
Having seen that you are always indenting error messages by 4, perhaps redefine verbose_error to be

     test -n "$verbose" && 1>&2 printf "    %s\n" "$@"

and ignoring my last comment.